aggregator/.env.example

```diff
@@ -1,21 +1,15 @@
-RPC_URL=http://localhost:8545
+RPC_URL=https://rinkeby.arbitrum.io/rpc
```

Comment:
I switched all of the `.env.example`'s to Rinkarby as I found this was the easiest/fastest way to get setup. We can change back if desired.